ass1/css/main.css

```diff
@@ -0,0 +1,160 @@
+body,
+html {
+  margin: 0;
+}
+
+* {
+  -webkit-font-smoothing: antialiased;
+  font-smoothing: antialiased;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+span,
+ul,
+li,
+a,
+button,
+input,
+select,
+textarea,
+table tr td {
+  margin: 0;
+  padding: 0;
+}
+
+em {
+  font-style: normal;
+}
+
+ul li {
+  list-style: none;
+}
+
+a {
+  color: #000;
+  cursor: pointer;
+  text-decoration: none;
+}
+
+h1 {font-size: 24px;}
+h2 {font-size: 22px;}
+h3 {font-size: 20px;}
+h4 {font-size: 18px;}
+h5 {font-size: 16px;}
+h6 {font-size: 14px;}
+p,
+ul li,
+a,
+button,
+input,
+label,
+select,
+textarea,
+table tr td {
+  font-size: 16px;
+}
+
+.clear {
+  clear: both;
+}
+
+.navbar {
+  background-color: #FA7207;
+  position: fixed;
+  text-align: center;
+  width: 100%;
+  z-index: 100;
+}
+
+.navbar-options li {
+  display: inline-block;
+}
+
+.menu {
+  color: #fff;
+  font-weight: 600;
+  line-height: 60px;
+  margin: 0 15px;
+  text-transform: uppercase;
+}
+
+.menu:hover {
+  color: #f9f9f9;
+}
+
+.container {
+  background-color: #fff;
+  padding: 6% 10px 2%;
```

Comment:
@priyanka0901 Why padding is both in pixel and percentage. Use either percentage or pixel.